mathesar_ui/src/pages/schemas/Schemas.svelte

```diff
@@ -0,0 +1,72 @@
+<script lang="ts">
+  import {
+    faPlus,
+    faTrash,
+    faPencilAlt,
+  } from '@fortawesome/free-solid-svg-icons';
+
+  import { schemas } from '@mathesar/stores/schemas';
+  import type { SchemaMap } from '@mathesar/stores/schemas';
+  import {
+    Icon,
+    Button,
+    SchemaRow,
+    TextInput,
+  } from '@mathesar-components';
+
+  // Prop
+  export let database: string;
```

Comment:
Our current UI considers only one database, when fetching schemas. This needs to be handled as part of https://github.com/centerofci/mathesar/issues/362.

Mentioning it here, to help keep track. No changes are required for it in this PR.